### ISSUE TITLE
Crashes in 1.08v

### ISSUE THREAD

**Issue by ac2cz** [ac2cz/FoxTelem#603]
![image](https://user-images.githubusercontent.com/14824793/70110894-82f2c680-161e-11ea-9bdb-5b7e505ae0c4.png)

Another
![image](https://user-images.githubusercontent.com/14824793/70110903-8b4b0180-161e-11ea-8455-4bfbd040a3f7.png)

And one seen by Alan from the AF system.  Two errors:
![image](https://user-images.githubusercontent.com/14824793/70110933-ab7ac080-161e-11ea-9a17-274e99fb6807.png)

![image](https://user-images.githubusercontent.com/14824793/70110946-b9304600-161e-11ea-92f4-1dc8d4a4cd6a.png)

**Comment by ac2cz** [comment]
And another crash from Larry:
![image](https://user-images.githubusercontent.com/14824793/70111002-ec72d500-161e-11ea-9103-d801cafc9c50.png)

**Comment by ac2cz** [comment]
Another from a FIAB:
![image](https://user-images.githubusercontent.com/14824793/70111025-01e7ff00-161f-11ea-8f49-8961d44e3038.png)

**Comment by ac2cz** [comment]
The errors in SatPayloadTable at line 201 in getPayloadData are due to a bug in loadSegments().  When we load in reverse order we tag every segment as recently accessed.  That prevents them being offloaded during the reset of the getPayloadData() method.  But when we load in normal order the called to tag the segments as accessed was missing.

I also noticed that the DELETE_LOCK was released by loadSegments() when we want it to stay on during the entire getPayloadData().  At first I thought that was the bug until I spotted this omission.  In fact with this corrected we should likely never have the issue and don't need the delete lock, but I will leave it in case a really really long load on a slow computer has a segment offloaded before it completed processing.

The FFT error is likely because the iqSource was null.  It was probably being switched or it had just started, so a race condition.  I set the method to return and do nothing if the iqSource is null.  This method tunes FindSignal and is called continually.

The null pointer in offloadSegments() is more of a puzzle.  It is likely another bug that has left a record as null.  We are reading the records to see if we can find the start of the records to offload.  We read one and it is null, so we crash.  I have set the code to ignore that as it is happening during the search, but it seems like there is another underlying issue..
